### ISSUE TITLE
Hello, slider doesn't support clicking?

### ISSUE THREAD

**Issue by Ctonys** [jeanregisser/react-native-slider#86]

**Comment by UI-Animation-Chen** [comment]
+1

**Comment by xuananpham93** [comment]
+1

**Comment by UI-Animation-Chen** [comment]
+1 again

**Comment by ajaswal** [comment]
This merge should suffice [#92](https://github.com/jeanregisser/react-native-slider/pull/92)

**Comment by arunkrishna39** [comment]
@ajaswal can you provide a sample for the above PR?

**Comment by DavitVosk** [comment]
PR is not merged yet ?

**Comment by OlivierFreyssinet** [comment]
+1

**Comment by zhaccc** [comment]
Would also like to see click support! +1

**Comment by zhaccc** [comment]
It seems its not mereged yet, PR has merge conflicts.

**Comment by jasongaare** [comment]
Yeah let's get some action on this!

**Comment by UI-Animation-Chen** [comment]
conflicts, squash it!!!

**Comment by GerardCasadevall** [comment]
+1

**Comment by amos80m** [comment]
any progress?

**Comment by Cassendra4** [comment]
any progress with the PR?

**Comment by fiznool** [comment]
I submitted a new PR, #136, with the conflicts fixed and the name changed (to 'pressable'). Hopefully it will be merged soon!

**Comment by bear-junior** [comment]
Hey, I have another question - is there any chance to make clickable button which changing value of slider? Something like 'plus' 'minus' button :)

**Comment by GerardCasadevall** [comment]
any progress with the PR?

**Comment by andresfcamacho** [comment]
Merge it! :)

**Comment by sjsikkema** [comment]
Any progress?

**Comment by illyax** [comment]
actually only after adding ``` this._fireChangeEvent('onValueChange');``` and the end of ```_handlePanResponderEnd``` I was able to get the pressed value.

**Comment by maciejpk** [comment]
any chances of merging it after over 1 year?
